ld/nodemcu.ld

```diff
@@ -95,7 +97,7 @@ SECTIONS
 
     _irom0_text_end = ABSOLUTE(.);
     _flash_used_end = ABSOLUTE(.);
-  } >irom0_0_seg :irom0_0_phdr
+  } >irom0_0_seg :irom0_0_phdr =0xffffffff
```

Comment:
Yes -- that is the fill value. Reduces stress on the flash. Especially when there is the sector used for the certificate.